refarch-frontend/Dockerfile

```
@@ -8,4 +8,4 @@
 COPY docker/nginx/*.conf "${NGINX_DEFAULT_CONF_PATH}"
 
 # Start the web server
-CMD nginx -g "daemon off;"
+CMD nginx -g "daemon off;"
```

Hadolint (2.12.0)

[warning] Use arguments JSON notation for CMD and ENTRYPOINT arguments

(DL3025)